exercises/acronym/example.rs

```diff
@@ -0,0 +1,20 @@
+pub fn abbreviate(phrase: &str) -> String {
+    phrase.split(|c: char| c.is_whitespace() || !c.is_alphanumeric())
+          .flat_map(|word| split_camel(word))
+          .filter(|word| !word.is_empty())
```

Comment:
You're totally right. I actually included that to deal with issues like `PHP: Hypertext Preprocessor`  where `.split()` would result in empty strings from having two characters I split on next to each other before I added `split_camel`. But since `split_camel` is in a flat map now and will return an empty list in the case of an empty string it effectively filters out those empty strings.

I think adding that using `.chars().next()` would be a good way to make it clear that we are handling the empty string case since the fact that `split_camel` dealt with it wasn't clear even to me. And I would like including `Option` handling just to reinforce the concept since it's so different from the `try catch` pattern.